File: hackerrank/problem solving/algorithms/1 - warmup/2 - simple array sum/with morfo library/simple_array_sum.java
// https://www.hackerrank.com/challenges/simple-array-sum/problem?isFullScreen=true
// Java 25

import morfo.io.Reader;
import morfo.math.Sum;

void main() {
    Reader.skipInputLn();
    IO.println(sumStdin());
}

// n: number of integers in the input line
// T: O(n)
// S: O(n) extra space
int sumStdin() {
    return Sum.ofInts(Reader.readLnInts());
}

File: hackerrank/problem solving/algorithms/1 - warmup/2 - simple array sum/with morfo library/simple_array_sum.cs
// https://www.hackerrank.com/challenges/simple-array-sum/problem?isFullScreen=true

using Morfo.IO;

Reader.SkipInputLn();
Console.WriteLine(SumStdin());

// n: number of integers in the input line
// T: O(n)
// S: O(n) extra space
int SumStdin() => Reader.ReadLn<int, List<int>>().Sum();
